kalite/settings/base.py

```diff
@@ -38,20 +38,76 @@
 # Basic setup of logging
 ##############################
 
-# TODO: Use Django's settings.LOGGERS
-
 # Set logging level based on the value of DEBUG (evaluates to 0 if False,
 # 1 if True)
 LOGGING_LEVEL = getattr(local_settings, "LOGGING_LEVEL", logging.INFO)
-LOG = getattr(local_settings, "LOG", logging.getLogger("kalite"))
 
-LOG.setLevel(LOGGING_LEVEL)
-ch = logging.StreamHandler()
-formatter = logging.Formatter('%(asctime)s - %(levelname)s:%(name)s: %(message)s')
-ch.setFormatter(formatter)
-LOG.addHandler(ch)
+# We should use local module level logging.getLogger
+LOG = getattr(local_settings, "LOG", logging.getLogger("kalite"))
 
-logging.getLogger("requests").setLevel(logging.WARNING)  # shut up requests!
+LOGGING = {
```

Comment:
All logging configuration is managed here. I welcome any further amendments to the configuration, but let's merge it first and refine it after. Right now, it works pretty well because we finally have verbose output of requests in `server.log`.